org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/AdapterUtils.java

```diff
@@ -67,4 +73,96 @@ public static String parseEnclosingType(String fullyQualifiedName) {
         }
         return null;
     }
+
+    /**
+     * Convert the source platform's line number to the target platform's line number.
+     *
+     * @param line
+     *           the line number from the source platform
+     * @param sourceLinesStartAt1
+     *           the source platform's line starts at 1 or not
+     * @param targetLinesStartAt1
+     *           the target platform's line starts at 1 or not
+     * @return the new line number
+     */
+    public static int formatLineNumber(int line, boolean sourceLinesStartAt1, boolean targetLinesStartAt1) {
```

Comment:
>formatLineNumber [](start = 22, length = 16)

minor, method name doesn't convey what the method does without reading the comments

Reply:
the same to formatPath

---
In reply to: [135172222](https://github.com/vscjavaci/eclipse.jdt.ls/pull/48#discussion_r135172222) [](ancestors = 135172222)